magpie/api/management/service/service_views.py

```diff
@@ -70,6 +70,7 @@ def register_service_view(request):
     service_url = ar.get_value_multiformat_post_checked(request, "service_url")
     service_type = ar.get_value_multiformat_post_checked(request, "service_type")
     service_push = asbool(ar.get_multiformat_post(request, "service_push"))
+    ax.verify_param(service_name, httpError=HTTPBadRequest, msgOnFail=s.Services_POST_BadRequestResponseSchema.description, isAlpha=True)
```

Comment:
Will indicate line too long during PR evaluation (other places also)
run `make lint` to validate any static code checks before push